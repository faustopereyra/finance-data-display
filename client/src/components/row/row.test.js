import { shallow } from "enzyme";
import React from "react";
import Row from "./row.component";

it("expect to render Row component", () => {
    expect(shallow(<Row />)).toMatchSnapshot();
})