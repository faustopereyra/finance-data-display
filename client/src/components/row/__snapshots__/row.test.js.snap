// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Row component 1`] = `ShallowWrapper {}`;
